azure-client-runtime/src/main/java/com/microsoft/azure/v2/CompletedPollStrategy.java

```diff
@@ -20,30 +20,29 @@
  * further polling.
  */
 public class CompletedPollStrategy extends PollStrategy {
-    private final HttpResponse bufferedOriginalHttpResponse;
```

Comment:
If it was important to say in the variable's name that it's buffered it seems worth it to express in the static type system that it's buffered. Might change this. A lot of these little changes in LROs were due to just clumsily trying to get it all working again after moving deserialization into HttpResponseDecoder.